React/Views/RCTSlider.m

```diff
@@ -32,4 +32,18 @@ - (void)setMaximumValue:(float)maximumValue
   super.value = _unclippedValue;
 }
 
+- (void)setThumbImage:(NSDictionary *) thumbImage
+{
+  UIImage *image = [UIImage imageNamed:thumbImage[@"source"]];
```

Comment:
See comments in https://github.com/facebook/react-native/pull/3850#issuecomment-153230107